back/spec/acceptance/ideas/ideas_create_spec.rb

```diff
@@ -136,8 +164,8 @@ def public_input_params(context)
             do_request
 
             assert_status 201
-            json_response = json_parse response_body
-            idea = Idea.find(json_response.dig(:data, :id))
+
+            idea = Idea.find(response_data[:id])
```

Comment:
The other changes in this file are just about simplifying the existing tests using helpers.